CHANGES.rst

```diff
@@ -62,6 +62,12 @@ Mosviz
 Specviz
 ^^^^^^^
 
+Specviz2D
```

Comment:
To be consistent with branding. But @duytnguyendtn should confirm since I joined after the rebranding occurred.
```suggestion
Specviz2d
```